en/intro/conventions.rst

```diff
@@ -172,6 +172,41 @@ automatically tied to the 'articles' table in the database), and renders to a
 file. None of these relationships have been configured by any means other than
 by creating classes and files that you'd need to create anyway.
 
++------------+-----------------------------------+---------------------------------------+------------------------------------------------------------------------+
+| Example    | articles                          | articles_categories                   |                                                                        |
++------------+-----------------------------------+---------------------------------------+------------------------------------------------------------------------+
+| DB         | articles                          | articles_categories                   | Tables names are plural and underscored.                               |
+|            |                                   |                                       | Field/Column names with two or more words are underscored: first_name  |     
++------------+-----------------------------------+---------------------------------------+------------------------------------------------------------------------+
+| File       | ArticlesController.php            | ArticlesCategoriesController          |                                                                        |
```

Comment:
Should ArticlesCategories have a .php here?

Reply:
adjusted